docs/fieldnorm/requirements_configuration.md

```diff
@@ -0,0 +1,28 @@
+# Requirements and configuration management
+
+The complexity and variety of developments in the field of bio-informatics, comes with a need for clear prerequisites before the start of a project on the one hand and concise regulation of development versioning on the other. Unfortunately, neither is described for application to medical devices by the ISO-15189, unlike the IEC-62304, which does disclose types of requirements that should be documented.
+
+## 1. Requirements in software development
+The requirements of software should be clear before starting on a project and describe the prerequisites that the software should meet for it to be considered as completed. The requirements may relate to the performance of the desired software, the physical characteristics, the computing environment, a need for compatibility, input- and output-structures, interfaces to other systems, communication from the software, security requirements, user interfaces to the software, data definitions, installation and acceptance requirements, requirements related to the methods of operation and maintenance, network-related requirements, user maintenance requirements and regulatory requirements.
+
+1.1. Forming requirements: All software development projects should have a standardized approach for gathering requirements. Requirements can come from end-users, bio-informaticians, or other stakeholders. It is essential to document these requirements clearly and track them throughout the development process. It is recommended to write the requirements in a form that is SMART (specific, measurable, attainable, realistic and time-bound), but this may not always apply to the situation.
```

Comment:
maybe move "It is recommended to write the requirements in a form that is SMART (specific, measurable, attainable, realistic and time-bound)" to point 1.2 instead of 1.1? or either remove 1.2 all together